app/models/card.rb

```diff
@@ -8,6 +8,39 @@ class Card < ActiveRecord::Base
   validates :list_id, presence: true, numericality: { only_integer: true }
   validates :title, presence: true, length: { minimum: 3 }
 
+  after_create :generate_tasks
+
+  def generate_tasks
+    # investigate better way to do this dynamically
+    tasks = ["Find advocate", "Contact advocate for meeting", "Apply to company", "Follow up with hiring manager about application", "Review for interview", "Send thank you email"]
+    tasks.each do |title|
+      self.tasks.create(title: title)
+    end
+  end
+
+  def set_next_task
+    task = self.tasks[list.position_id]
+    self.update(next_task: task.title)
+  end
+
+  def recalculate_priority
```

Comment:
Seems like a service and/or an enum would have been useful here. Maybe a state machine?